bootstrap.sh

```diff
@@ -71,15 +71,15 @@ mkdir -p install
 
 cd deps
 
-lwgrp=lwgrp-1.0.5
-dtcmp=dtcmp-1.1.4
-pdsh=pdsh-2.34
+lwgrp=lwgrp-1.0.6
+dtcmp=dtcmp-1.1.5
+pdsh=pdsh-2.35
```

Comment:
Did all of these sub-components change?

Reply:
There was a recent small release to lwgrp (I don't believe anything changed in terms of how it works).
Both DTCMP and pdsh had releases some time ago, but didn't get updated in bootstrap.